issues/Issues.md

```diff
@@ -0,0 +1,53 @@
+# Windows Issues
+
+## /dev/null git apply issue
+
+New version of git is required to solve this issue.
```

Comment:
Minimum required version is set to 2.19.0

Reply:
Cool, I was just wondering how to easily install cmake 3.12 on Ubuntu, since apt install only install 3.10 on Ubuntu 18.

Reply:
This is git version. Last comment was wrong.

Reply:
Minimum required version of git is set to 2.19.0 in cmake. It is a check that has been added in CMakeLists.txt

Reply:
Has modified.